### DISCUSSION TITLE
Song title font is too small.

### DISCUSSION THREAD

**Discussion by HubKing** [UnicornsOnLSD/finamp#349] [General]
I mean the font like the song title, time, musician name in the SONGS tab or the album details list (this is my arbitrary name. I mean the page that appears when you click an album in the ALBUMS tab). The font is relatively smaller than that of the fonts of other tabs. Is this intentional?

If it is intentional, can we have the customisation option for that font size? If this is because of limited space, I would rather hide the thumbnail of the song, especially in the album details list, where all songs have the same thumbnails anyway.

**Comment by Chaphasilor** [comment]
Are you talking about the first or second line of the song item? Because the first line (song title) seems to be the exact same size as the name of playlists or albums.

The text on the second line is smaller, like the artist name for songs or albums.

A screenshot of what you mean would be nice :)

**Comment by HubKing** [comment]
See the screenshots below. The fonts of other tabs (the column) is bigger than the song title of the SONGS tab (top-right). Compare the song title font with that of another music player Musicolet (bottom-right).

![image](https://user-images.githubusercontent.com/10900489/195364967-d59ae478-686f-46a0-b316-2b11c6b05078.png)

**Comment by Chaphasilor** [comment]
Hmm, which version of the app are you using? And what phone?

This is how it looks for me:

![Screenshot_20221012-180018_Finamp~2.png](https://user-images.githubusercontent.com/18015147/195393134-15ee5c25-6d54-4169-9819-a56ea9a70544.png)

![Screenshot_20221012-180028_Finamp~2.png](https://user-images.githubusercontent.com/18015147/195393190-44ce77d7-14cc-48c1-bcad-579264807b08.png)

![Screenshot_20221012-180355_Finamp.png](https://user-images.githubusercontent.com/18015147/195393167-b01b947e-25d7-4bcd-b715-288556dd7e48.png)

**Reply by HubKing** [reply]
It seems that only the SONGS tab does not respect system font settings. Open Settings -> Display -> Font size and change "Default" to "Largest". IF I use "Default", then yes, the font on the SONGS tab looks the same size as other tabs. But if I change it to "Largest", other tabs' fonts becomes larger, but that of the SONGS tab remains the same.

**Reply by Chaphasilor** [reply]
Well there you go, bug identified :D

You should open an issue about it, looks like an easy fix :)

**Reply by HubKing** [reply]
Created https://github.com/jmshrv/finamp/issues/350
